### ISSUE TITLE
Child reactions of child reactions do not get updated properly

### ISSUE THREAD

**Issue by arnautov-anton** [GetStream/react-activity-feed#304]
When user creates a reply comment to a parent reaction and then right after tries to create another child reaction to that newly created child reaction (of kind `like` in this instance) - the `own_children` key (and basically other keys also) of that object won't get updated properly. The application crashes while trying to find path with `FeedManager.getReactionPaths` which returns `undefined` in `FeedManager.onToggleChildReaction` or later in `FeedManager.onAddChildReaction`.

https://user-images.githubusercontent.com/43254280/134659756-bd996ccc-422e-48a9-9a64-742643cc8256.mov

**Comment by antishok** [comment]
Hi, we're seeing the same, or a very similar issue.
When replying to a comment, `FeedManager.getReactionPaths` is called on an `activity` param which is actually the comment object (the comment to which we're replying) and returns undefined, causing an error.
Any updates on this? Thanks

**Comment by antishok** [comment]
In case it helps, this is the gist of what we're doing:
```jsx
<CommentList
  activityId={someActivityId}
  CommentItem={(comment_props) => (
    ...
    <CommentField activity={comment_props.comment} />
  )}
/>
```

**Comment by arnautov-anton** [comment]
Hey, @antishok! Sadly, no update yet, but the reply functionality (activity -> comment -> reply comment) is unaffected by this. It's the children of reply comments (likes or other sub-reply comments) that cause this issue, which is also a maximal nesting depth allowed by the BE.

**Comment by antishok** [comment]
Thanks for the reply @arnautov-anton.
That's odd then because we _are_ having this issue with just replies to comments (to activities),
but maybe we are doing something wrong on our end.

Can you please confirm that the usage I showed above is correct?  
Specifically this part:  `<CommentField activity={comment_props.comment} />` 
Is it correct to pass the comment object as the activity to `CommentField`?
The documentation has almost no info on how to properly do replies to comments.

Thanks again!

**Comment by arnautov-anton** [comment]
Sure! Here's a [simple example](https://gist.github.com/arnautov-anton/ddad4302d8f13a2c2037d804ccfeafb7) I've been using for testing - works fine without the likes on the replies. Yours seems to be pretty close to how it's used normally but feel free to try it out and let me know if you're still experiencing the issues.

Also I'm sorry about the missing documentation - we're currently working on it and we plan to release it as soon as it's finished. Sorry for the inconvenience.

**Comment by antishok** [comment]
Thanks for the example. 
But it seems pretty different, you are not using `CommentField`, but using `useFeedContext` - what is that? Is it documented somewhere?

**Comment by antishok** [comment]
Figured it out - and your method indeed seems to work, so thanks a lot.
Unfortunately it requires re-implementing the `CommentField` component, and using undocumented API's :)

**Comment by antishok** [comment]
I've modified `CommentField` to work with comment replies,
I'll share it here in case it's useful to someone: [CommentReplyField.jsx](https://gist.github.com/antishok/6a0f5e85bfbae615d360fdff8c167c9f)

**Comment by arnautov-anton** [comment]
Hey again, @antishok! I missed your point at first but I understand now, I'll create a separate issue to extend current `CommentField` component to accept `comment` as well as `activity` object so it can be used with comment replies. Thank you for pointing this out and thank you for your contribution. :slightly_smiling_face:

**Comment by DmitriyNikolenko** [comment]
Facing the same issue, when we "created comment -> replied to comment => liked comment reply - > error is thrown".
An error is thrown because FeedManager.getReactionPaths is 'undefined'. Only after a refresh of the SinglePost a user can like a reply.

**Comment by therajumandapati** [comment]
Hi, are there any plans to fix this issue?
